.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - feature/cek/release-workflow
+    tags:
+      - "*"
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit
+        uses: actions/checkout@v2
+
+      - name: Install correct version of Node
+        shell: bash -l {0}
+        run: nvm install
+
+      - name: Set up registry authentication
+        uses: actions/setup-node@v1
+        with:
+          registry-url: https://registry.npmjs.org
+
+      - name: Install correct version of Yarn
+        run: yarn policies set-version $(cat .yvmrc)
+
+      - name: Install release dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Build and publish package
+        run: npm publish --dry-run
```

Comment:
This is here for testing and will output the files that will be included in the final package. Yarn unfortunately lacks a `--dry-run` option for its publish command. I'd like to perform the actual release post-merge to exercise the release instructions included in an issue template. I have a high degree of confidence that `yarn publish` will just work, but I can try to create a test case to increase our confidence if necessary.